tests/examples/safe_remote_purchase/test_safe_remote_purchase.py

```diff
@@ -7,86 +7,93 @@
 # 3. Seller ships item
 # 4. Buyer confirms receiving the item. Buyer's deposit (value) is returned. Seller's deposit (2*value) + items value is returned. Balance is 0.
 import pytest
-from ethereum.tools import tester
-from ethereum import utils
 
-contract_code = open("examples/safe_remote_purchase/safe_remote_purchase.v.py").read()
+
 # Inital balance of accounts
-INIT_BAL = 1000000000000000000000000
+INIT_BAL_a0 = 1000000000000000000000000
+INIT_BAL_a1 = 1000000000000000000000000
+
 
+def ETH(x):
+    return x * 10**18
```

Comment:
In case you aren't aware, there are nice APIs for this in both `eth-utils` and web3

- `eth_utils.to_wei` and `eth_utils.from_wei`
- `w3.toWei` and `w3.fromWei`

Reply:
Ah yes, I remember now! :smile: